constant.go

```diff
@@ -8,10 +10,10 @@ const (
 	MaxLogChunkSize uint64 = 1.5 * (1024 * 1024)
 
 	// NormalSendSpeedInterval is a bulk send interval in normal mode
-	NormalSendSpeedInterval float64 = 0.5
+	NormalSendSpeedInterval = 1 * time.Second
```

Comment:
also changed this... when sending in bulks, 1 second (x2 from 500ms) is also fine